jjb/edgex-go/edgex-go-snap.yaml

```diff
@@ -12,6 +12,10 @@
           branch: 'delhi'
           build_script: 'cd snap && ./build.sh'
           snap-channel: delhi/edge
+      - 'edinburgh':
+          branch: 'edinburgh'
+          build_script: 'cd snap && ./build.sh'
```

Comment:
This is wrong, we only override `build_script` for delhi, it can be left as-is for edinburgh.

Reply:
@anonymouse64 `build_script` line removed.